src/compute-client/src/controller.rs

```diff
@@ -452,6 +455,17 @@ where
         }
     }
 
+    /// Update compute configuration.
+    pub fn update_configuration(&mut self, config_params: BTreeSet<ComputeParameter>) {
+        for instance in self.instances.values_mut() {
+            instance.update_configuration(config_params.clone());
+        }
```

Comment:
Do we need to filter out linked clusters here?

Reply:
Nope! The compute control doesn't know about linked clusters (adapter never calls `create_instance` for those), so it ignores them automatically.